catroid/src/main/java/org/catrobat/catroid/ui/MainMenuActivity.java

```diff
@@ -191,16 +188,7 @@ public void onPause() {
 	@Override
 	public boolean onCreateOptionsMenu(Menu menu) {
 		getMenuInflater().inflate(R.menu.menu_main_menu, menu);
-
```

Comment:
I know I told you to remove this but can you please reset the MainMenuActivity to the develop's state? we still need that.

Reply:
ok changed this + the colors.xml is now also on develop state, previously it didn't have the beta_color.